docker-compose.yml

```diff
@@ -15,12 +15,32 @@ services:
       SYNDICATION_API_KEY: ${SYNDICATION_API_KEY}
       SYNDICATION_SERVICE_END_POINT: http://v1.syndication.nhschoices.nhs.uk/services/types/sexualhealthinformationandsupport
```

Comment:
Should this have a default now or could it just be `${SYNDICATION_SERVICE_END_POINT}`?
Same goes for `ETL_NAME` above.

Reply:
I've removed the defaults from here and explicitly set the required one in the docker-compose-test.yml.